certificate-policy.md

```diff
@@ -411,25 +416,29 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 
 # 2. PUBLICATION AND REPOSITORY RESPONSIBILITIES
-The Federal PKI Policy Authority SHALL annually update this CP to ensure compliance with CAB Forum Baseline requirements.
+The Federal PKI Policy Authority SHALL review and update this CP at least every 365 days to ensure compliance with CAB Forum Baseline requirements.
```

Comment:
Same comment as above.

Reply:
updated and moved everything into 2.0 for publication of CP, CPS, timeframes, etc. 

https://github.com/uspki/policies/pull/147/commits/60c2d521b1bcbae27540118e060d870e50019c50